### ISSUE TITLE
SyntaxError in create_new_username

### ISSUE THREAD

**Issue by marcusklaas** [quentinsf/qhue#10]
Hi Quentin,

First of all: thanks a bunch for creating and maintaining this library. It looks to be exactly what I need.

I've been playing around with it a bit and ran into the following error when creating a new username:
```
Press the Bridge button, then press Return: 
Traceback (most recent call last):
  File "phone-checker.py", line 13, in <module>
    username = create_new_username(hue_bridge_ip)
  File "/home/marcusklaas/.local/lib/python2.7/site-packages/qhue/qhue.py", line 71, in create_new_username
    real_raw_input("Press the Bridge button, then press Return: ")
  File "<string>", line 0
    
    ^
SyntaxError: unexpected EOF while parsing
```
I've basically just run your example code on python 2.7.11 on Ubuntu 16.04. 

When I replaced your python 3 compatibility hack by a simple `raw_input` call, it started working for me..

Probably an easy fix, but I'm too novice to cook something up that should work for both python 2 and 3 and most common platforms...

**Comment by quentinsf** [comment]
Sorry Marcus - I'm not ignoring this - just a bit swamped...  Will fix soon!

**Comment by edwardvdv** [comment]
I can confirm that the proposed fix by marcusklaas works with Python 2.7.11 on Windows.

To elaborate: comment out line 70 in qhue.py:
# real_raw_input = getattr(__builtins__, 'raw_input', input)

and replace real_raw_input with raw_input in line 71.

Cheers.

**Comment by quentinsf** [comment]
OK, I think version 1.0.3 should have a more reliable fix for this - am closing, but re-open if you still see any issues.  Thanks all.
